### PR TITLE
Docs: Improve loadbalancer documentation.

### PR DESCRIPTION
**Pull request by panuhorsmalahti**
Improve loadbalancer docs for clarity.

### CODE REVIEW

File: docs/using-kontena/loadbalancer.md [hunk 1]
```diff
@@ -231,7 +231,7 @@ These options are defined on the services that are balanced through Kontena Load
 * `KONTENA_LB_INTERNAL_PORT`: service port that is attached to load balancer
 * `KONTENA_LB_EXTERNAL_PORT`: service port that load balancer starts to listen (only for tcp mode)
 * `KONTENA_LB_VIRTUAL_HOSTS`: comma-separated list of virtual hosts, if you would like to access your service through the vpn, you'll need to add that as a virtual host as well (only for http mode)
-* `KONTENA_LB_VIRTUAL_PATH`: path that is used to match request; example: "/api" (only for http mode)
-* `KONTENA_LB_KEEP_VIRTUAL_PATH`: if set to true, virtual path will be kept in request path (only for http mode)
+* `KONTENA_LB_VIRTUAL_PATH`: path that is used to match request; example: "/api" (only for http mode). By default, a request to e.g. `loadbalancer/api/foo` is proxied to e.g. `service/foo`
```
Comment (jnummelin) [inline docs/using-kontena/loadbalancer.md]: Yes, but now `loadbalancer` and `service` gets now easily confused as being part of the path. Maybe something like:
By default the path is omitted from the forwarded request, so a request to path `/api/foo` is proxied with path `/foo`

File: docs/using-kontena/loadbalancer.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ services:
 These environment variables configure the load balancer itself.
 
 * `KONTENA_LB_HEALTH_URI` - URI at which to enable Kontena Load Balancer level health check endpoint. Returns `200 OK` when Kontena Load Balancer is functional.
-* `STATS_PASSWORD` - The password for accessing Kontena Load Balancer statistics.
+* `STATS_PASSWORD` - The password for accessing Kontena Load Balancer statistics using HTTP Basic authentication. The default username is `stats`.
```
Comment (jnummelin) [inline docs/using-kontena/loadbalancer.md]: Add also the default passwd: `secret`:)

File: docs/using-kontena/loadbalancer.md [hunk 3]
```diff
@@ -103,7 +103,7 @@ See HAProxy [docs](http://cbonte.github.io/haproxy-dconv/1.6/configuration.html#
 
 ## Stats
 
-Kontena Load Balancer exposes a statistics web UI only on the private overlay network. To access the statistics you must use the [VPN](vpn-access.md) to access the overlay network. The statistics are exposed on port 1000 of the Kontena Load Balancer instances.
+Kontena Load Balancer exposes a statistics web UI only on the private overlay network. To access the statistics you must use the [VPN](vpn-access.md) to access the overlay network. The statistics are exposed on port 1000 of the Kontena Load Balancer instances. You can also expose the port 1000 using the ports option in `kontena.yml`.
```
Comment (jnummelin) [inline docs/using-kontena/loadbalancer.md]: Maybe add a note that if exposing the stats to public networks one should really override the default passwd with strong password.

File: docs/using-kontena/loadbalancer.md (hunk 2 above)
Comment (panuhorsmalahti) [inline docs/using-kontena/loadbalancer.md]: Added

File: docs/using-kontena/loadbalancer.md (hunk 3 above)
Comment (panuhorsmalahti) [inline docs/using-kontena/loadbalancer.md]: Added

File: docs/using-kontena/loadbalancer.md (hunk 1 above)
Comment (panuhorsmalahti) [inline docs/using-kontena/loadbalancer.md]: Changed

**Pull request merged**